__tests__/endpoints.test.js

```diff
@@ -239,3 +239,42 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("Status 200 - should return an array of comments via article id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toEqual({

```

Comment:
we should expect that each comment has the article_id that we are requesting also

Reply:
Aplogies. The task doesn't state to include it so just assumed it didn't want it